src/architecture/msgPayloadDefC/ConstDynEffectorMsgPayload.h

```diff
@@ -1,7 +1,7 @@
 /*
  ISC License
 
- Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+ Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
This message header changes should be split from this commit and squashed into the first commit.

Reply:
Resolved